src/Workspaces/CSharp/Portable/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -234,12 +242,15 @@ public static CSharpSyntaxContext CreateContext(Workspace workspace, SemanticMod
                 syntaxTree.IsInstanceContext(position, leftToken, cancellationToken),
                 syntaxTree.IsCrefContext(position, cancellationToken) && !leftToken.IsKind(SyntaxKind.DotToken),
                 syntaxTree.IsCatchFilterContext(position, leftToken),
-                isDestructorTypeContext);
+                isDestructorTypeContext,
+                inferredTypes);
         }
 
         public static CSharpSyntaxContext CreateContext_Test(SemanticModel semanticModel, int position, CancellationToken cancellationToken)
         {
-            return CreateContext(/*workspace*/null, semanticModel, position, cancellationToken);
+            var inferenceService = new CSharpTypeInferenceService();
```

Comment:
This method is used from the keyword recommender test which don't have a workspace or composition; figured it was easier to just new one up for test purposes.